articles/openshift/delete-cluster.md

```diff
@@ -35,16 +35,18 @@ In previous articles for [creating](create-cluster.md) and [connecting](connect-
 
```

Comment:
Hi @ehvs.  These variables don't seem to track with the pages referenced. In the Create doc, the variables were set as follows:

RESOURCEGROUP=aro-rg
CLUSTER=cluster

Should your corrections not track with these names?

Reply:
Hi John, I didnt change the resourcegroup value from what it was before, but I think being generic as is should be fine. Since these variables can use any name specified by the user.